reana_server/rest/workflows.py

```diff
@@ -1488,28 +1495,30 @@ def open_interactive_session(workflow_id_or_name):  # noqa
           examples:
             application/json:
               {
-                "message": "Workflow 256b25f4-4cfb-4684-b7a8-73872ef455a1 does
-                            not exist."
+                "message": "Interactive session type jupiter not found, try
+                            with one of: [jupyter]."
               }
         500:
           description: >-
             Request failed. Internal controller error.
     """
     try:
         user_id = get_user_from_token(request.args.get('access_token'))
-
+        if interactive_session_type not in INTERACTIVE_SESSION_TYPES:
```

Comment:
This check could be also done on the client side, to avoid unnecessary calls.

Reply:
It is already done [here](https://github.com/reanahub/reana-client/pull/257/files#diff-19bed1e7334d00a24f059e517c55f01dR887) with Click choices :)